tests/terraform-eks/create/aws-eks.tf

```diff
@@ -0,0 +1,33 @@
+module "eks" {
```

Comment:
shouldn't here be module version set also?

Reply:
Kubernetes version set as a variable.